services_app/build.gradle

```diff
@@ -122,6 +122,7 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'androidx.preference:preference:1.1.1'
     implementation 'androidx.fragment:fragment:1.3.0-beta01'
+    implementation 'com.android.support:design:28.0.0'
```

Comment:
Please update this library, see https://material.io/develop/android/docs/getting-started#new-namespace-and-androidx